org.mozilla.Thunderbird.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "org.mozilla.Thunderbird",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "thunderbird",
+    "rename-icon": "thunderbird",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Access to thunderbird configuration and cache files */
+        "--filesystem=~/.cache/thunderbird", "--filesystem=~/.thunderbird",
+        /* Access to home directory (for attachments) */
+        "--filesystem=home",
```

Comment:
Should I make this `--filesystem=home:ro` to make access read-only?

Reply:
If write access isn't needed yes.

Reply:
I wonder. Can the home be made read-only (so the user can upload attachments), but `xdg-download` be exposed as read-write via `--filesystem=xdg-download` so the user can download files?

Reply:
Yes, you can expose both of them on different lines and it'll work as expected. People may get slightly annoyed, but they'll get over it.

Reply:
Works perfectly, change has been made.